_data/developer.yml

```diff
@@ -114,6 +118,7 @@ websites:
     - name: Docker
       url: https://docker.com/
```

Comment:
Their current URL is now `https://www.docker.com/`. Could you please update the `url` to reflect this?